qemu_v8.mk

```diff
@@ -155,6 +157,10 @@ linux: linux-common
 	mkdir -p $(BINARIES_PATH)
 	ln -sf $(LINUX_PATH)/arch/arm64/boot/Image $(BINARIES_PATH)
 
+linux-modules: linux
+	$(MAKE) -C $(LINUX_PATH) $(LINUX_COMMON_FLAGS) modules
+	$(MAKE) -C $(LINUX_PATH) $(LINUX_COMMON_FLAGS) INSTALL_MOD_STRIP=1 INSTALL_MOD_PATH=$(MODULE_OUTPUT) modules_install
```

Comment:
It is slightly annoying to have `Image` in `$(LINUX_COMMON_FLAGS)` because it is repeated, but that's OK.